src/lib.rs

```diff
@@ -507,7 +511,7 @@ impl<'a> AggregatedQueries<'a> {
         let language = language.language();
         for (rule_index, instantiated_rule) in instantiated_rules.into_iter().enumerate() {
             for (rule_listener_index, rule_listener_query) in instantiated_rule
-                .rule
+                .rule_instance
```

Comment:
Per comment on previous PR, it did seem like the "listener query" retrievals wanted to be off of the `RuleInstance` (vs the `Rule`) so moved `.listener_queries()`